pkg/v1/tkg/client/cluster.go

```diff
@@ -171,7 +171,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 	}
 
 	log.Infof("Creating workload cluster '%s'...", options.ClusterName)
-	err = c.DoCreateCluster(regionalClusterClient, options.ClusterName, options.TargetNamespace, string(bytes))
+	//err = c.DoCreateCluster(regionalClusterClient, options.ClusterName, options.TargetNamespace, string(bytes))
```

Comment:
Is this intended?

Reply:
PR was not ready to review, it is ready now you can review it again.